tango_simlib/utilities/base_parser.py

```diff
@@ -2,17 +2,21 @@
 from __future__ import division
 from __future__ import absolute_import
 
+
 #########################################################################################
 # Copyright 2018 SKA South Africa (http://ska.ac.za/)                                   #
 #                                                                                       #
 # BSD license - see LICENSE.txt for details                                             #
 #########################################################################################
-import abc
+from future import standard_library
+standard_library.install_aliases()
 
+from builtins import object
+import abc
+from future.utils import with_metaclass
```

Comment:
Please reorder the imports as per [CAM - Python style guide](https://docs.google.com/document/d/1aZoIyR9tz5rCWr2qJKuMTmKp2IzHlFjrCFrpDDHFypM/edit#heading=h.y0sdkqkmrbb3)